contrib/coredns/policy/policy.go

```diff
@@ -92,6 +93,45 @@ func (p *policyPlugin) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dn
 		p.attrPool.Put(attrBuffer)
 	}()
 
+	if p.conf.ownIPEndpoint != "" {
+		var buff bytes.Buffer
+		buff.Write([]byte(p.conf.ownIPEndpoint))
+		buff.Write([]byte("."))
+		buff.Write([]byte(p.conf.debugSuffix))
```

Comment:
Any better way to do this, maybe?

Reply:
concatenate it once (in parsing code) then store in config